acbuild/README.md

```diff
@@ -13,3 +13,13 @@ Binary package
 ## Usage
 
 *TODO: Add instructions for usage*
+
+## Testing
+
+Run the BATS tests for this plan against a package ident like so:
+
+```bash
+hab pkg build acbuild
```

Comment:
Should this be `hab studio build`?

Reply:
I thought `pkg build` would open a studio for itself. When I try both `pkg build` and `studio build` on my laptop, the output appears to be identical.
Is there a difference?

The only difference I see is that on Windows if you want to ensure a docker studio is used, you need `hab studio -D build` since pkg doesn't accept that option.

Reply:
@smacfarlane mentioned that `pkg build` is preferred and `studio build` is an alias that may be deprecated.

Reply:
I took a deeper look here and I discovered a couple of things:

1. There's no indication that the `hab studio build` command is deprecated in the code.
2. The `pkg build` method does use Clap's more modern way of managing the CLI.

So I think the argument here only lightly favors `hab pkg build`.

In any case, this issue should not block this PR. There are already efforts underway to create a single binary to reduce this confusion in the future.

https://github.com/habitat-sh/habitat/issues/5233